### ISSUE TITLE
intermittent api.sock: bind: address already in use

### ISSUE THREAD

**Issue by AkihiroSuda** [rootless-containers/rootlesskit#30]
```
+ rootlesskit --state-dir /run/user/1001/usernetes/rootlesskit --net=slirp4netns --mtu=65520 --copy-up=/etc --copy-up=/run --copy-up=/var/lib ./run.sh default-crio
[rootlesskit:parent] error: listen unix /run/user/1001/usernetes/rootlesskit/api.sock: bind: address already in use
```

**Comment by u5surf** [comment]
Hi, I think it works better to be unlinking the unix domain socket file before listening
in pkg/parent/parent.go
https://github.com/rootless-containers/rootlesskit/blob/24449198253bb74f38d5c1a6270196f58f2871da/pkg/parent/parent.go#L247
ref:
https://troydhanson.github.io/network/Unix_domain_sockets.html

**Comment by AkihiroSuda** [comment]
Thanks, if you don't mind could you open PR?

**Comment by AkihiroSuda** [comment]
Fixed in #31
